kernel/kernel-api/src/main/java/io/delta/kernel/expressions/Column.java

```diff
@@ -47,6 +60,6 @@ public List<Expression> getChildren() {
 
     @Override
     public String toString() {
-        return "column(" + name + ")";
+        return "column(" + Arrays.stream(names).collect(Collectors.joining(".")) + ")";
```

Comment:
what about column names that already have a `.` inside of them?

e.g. `names = [a, b.c, d]` ?

Should we detect this and escape with back ticks?

i.e. should the `toString` be:
```
a.b.c.d
or
a.`b.c`.d
```

Reply:
Changed to generate: 
```
`a`.`b.c`.`d`
```